.gitlab/pipeline/build.yml

```diff
@@ -329,6 +329,14 @@ build-linux-substrate:
     # - printf '\n# building node-template\n\n'
     # - ./scripts/ci/node-template-release.sh ./artifacts/substrate/substrate-node-template.tar.gz
 
+build-minimal-runtime-polkavm:
+  stage: build
+  extends:
+    - .docker-env
+    - .common-refs
+  script:
+    - SUBSTRATE_RUNTIME_TARGET=riscv cargo check -p minimal-runtime
```

Comment:
Btw why does it only run check and not build?  
I think there are possible errors that only arise on `build`.

Reply:
The builder runs as part of the `build.rs`. It can not detect if you are running `cargo check` and thus, it will always compile. So, everything works here as expected.